image/exposure/driver.py

```diff
@@ -0,0 +1,82 @@
+import torch
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.